common/utils.go

```diff
@@ -158,3 +159,12 @@ func TruncateWithDot(s string) string {
 func Ptr[T any](v T) *T {
 	return &v
 }
+
+func ContainsAddress(addresses []types.Address, address types.Address) bool {
```

Comment:
https://pkg.go.dev/slices#Contains

Reply:
I asked AI if there was a function to do that and it said no :unamused: Stupid clankers